### DISCUSSION TITLE
Use of uninitialized value in pattern match (m//) at /home/koyomi/lanraragi/script/../lib/LANraragi/Model/Search.pm line 431.

### DISCUSSION THREAD

**Discussion by junpet** [Difegue/LANraragi#1016] [Q&A]
I use docker version on linux. In the latest version (v.0.9.1) the thumbnail folder has to be declared explicitly. I created a new folder, added to the volumes, regenerated all of the thumbnails, but since then the following error can be seen in the logs and there are no thumbs for the newly uploaded archives.

**Comment by Difegue** [comment]
I don't think this error is related to thumbnails, looks just like a warning due to empty strings ending up in search or something.  

Does your new thumbnail folder have anything in it after regenerating everything?

**Reply by junpet** [reply]
Sorry for late answer, got busy. Thank your answer. Can you help me what could cause that the thumbnails are not generated for newly uploaded archives? Using the `Generate Missing Thumbnails` button generates the missing thumbnails, but the following notification shows up: 
![image](https://github.com/user-attachments/assets/fbf2328d-58e1-469a-8356-68a3175cbb28)
Checked the logs of the container and the application logs, but there's no errors.
